src/resources/views/base/inc/scripts.blade.php

```diff
@@ -1,6 +1,18 @@
+@basset('https://unpkg.com/jquery@3.6.1/dist/jquery.min.js')
+@basset('https://unpkg.com/popper.js@1.16.1/dist/umd/popper.min.js')
+@basset('https://unpkg.com/noty@3.2.0-beta-deprecated/lib/noty.min.js')
+@basset('https://unpkg.com/pace-js@1.2.4/pace.min.js')
+@basset('https://unpkg.com/sweetalert@2.1.2/dist/sweetalert.min.js')
+
+@include('backpack::inc.theme_scripts')
+
 @if (config('backpack.base.scripts') && count(config('backpack.base.scripts')))
     @foreach (config('backpack.base.scripts') as $path)
-    <script type="text/javascript" src="{{ asset($path).'?v='.config('backpack.base.cachebusting_string') }}"></script>
+        @if(is_array($path))
+            @basset(...$path)
+        @else
+            @basset($path)
+        @endif
     @endforeach
 @endif
```

Comment:
@promatik I noticed in the form... we get a nasty console error `No method named "destroy"` every time we:
- switch tabs
- use a field (any field)

This didn't happen before we used bassets. Everything still works, sure, but... it's annoying and we should fix it ASAP. Can you please get to the root cause of this and fix it? The sooner we do this the better, otherwise we'll keep cramming more things into this branch and there will be more things that might have caused it.

![CleanShot 2023-02-21 at 10 36 51](https://user-images.githubusercontent.com/1032474/220291628-a723ff6b-ba2a-4923-ae2d-c7af0497678b.png)

Reply:
Now that I think about it... it's probably related to our CrudField JS API... otherwise... why would it happen every time we use a field?